changelog/3615.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed bugs in `remote_data_manager.py` which fixes bold type and extra line
+breaks in remote data manager.
```

Comment:
This is not a bug fix but a documentation change, could you reword the changelog and the type of the change please?